entries/jQuery.xml

```diff
@@ -58,6 +58,7 @@ $( "div.foo" ).click(function() {
 
       <h4 id="using-dom-elements">Using DOM elements</h4>
       <p>The second and third formulations of this function create a jQuery object using one or more DOM elements that were already selected in some other way. When passing an array, each element <em>must</em> be a DOM element; mixed data is not supported. A jQuery object is created from the array elements in the order they appeared in the array; unlike most other multi-element jQuery operations, the elements are not sorted in DOM order.</p>
+      <p>Please note that you can pass text nodes into a jQuery collection this way, although, the default assumption should be that jQuery doesn't support operations on text nodes. The few APIs that support text nodes will have an explicit note about that on their API documentation page.</p>
```

Comment:
@gibson042 : Take a look at https://github.com/jquery/api.jquery.com/issues/149. The part that begins with "Dave," I believe, is referring to @dmethvin.